c2rust-analyze/src/context.rs

```diff
@@ -834,8 +834,12 @@ impl<'tcx> GlobalAnalysisCtxt<'tcx> {
         self.fn_origins = fn_origin_args_params(self.tcx, &self.adt_metadata);
     }
 
-    pub fn function_context<'a>(&'a mut self, mir: &'a Body<'tcx>) -> AnalysisCtxt<'a, 'tcx> {
-        AnalysisCtxt::new(self, mir)
+    pub fn function_context<'a>(
+        &'a mut self,
+        mir: &'a Body<'tcx>,
+        base: u32,
```

Comment:
Should the name be more detailed, e.g., `ptr_base`?

Reply:
Changed both `base` arguments to `local_ptr_base` (b68e56876)